src/scss/base.scss

```diff
@@ -16,7 +16,7 @@ html, body {
 body {
     color: $text-color;
     background-color: $background-color-alt2;
-
+    -webkit-app-region: drag;
```

Comment:
Draggable styling for macos is defined in `environment.scss`. Changes should go there. We need these to be defined on elements explicitly to avoid weird behaviors around dragging.

Reply:
Thanks, I see that now. [This](https://github.com/bitwarden/desktop/blob/7499d7de2522fab2c05f02966efe1eb45c196f95/src/app/vault/vault.component.ts#L187) line of code is causing the body to lose the class defined for this when the vault loads, which is making the app lose all it's draggability. Any idea why this class the layout_frontend class is being removed here @kspearrin ?

Reply:
That class is for "frontend" layouts (i.e. the layout on pages when you are not logged in). The draggable stuff was intentionally not made available on all areas of the window since there is a lot more stuff included on-screen whenever you log in. Usually, there is some kind of defined bar at the top or something that you use as a draggable area.

Reply:
For the mac desktop app, there isn't much. Draggable areas are limited to the spots outlined here. If that's the intended design we can leave this as is and I'll close the linked ticket. Let me know. 

<img width="1773" alt="Screen Shot 2020-10-01 at 12 34 04 PM" src="https://user-images.githubusercontent.com/15897251/94837728-9f989000-03e2-11eb-96bb-024997cac2e6.png">

Reply:
That is the intention today. I agree that it would be nice to have a larger area that is draggable, but I am not sure how to make that work with this design layout. We don't want to just all background areas draggable.